autocontingency.lic

```diff
@@ -3,9 +3,6 @@
 =end
 custom_require.call(%w(common common-arcana events spellmonitor drinfomon common-travel))
```

Comment:
Can `common` be removed? I don't see where it is used...

Reply:
I think we can remove `events` and `drinfomon` too since `spellmonitor` will depends on `drinfomon` anyway.

Reply:
I am sure we can remove drinfomon here, because  of spell monitor. 

Events looks like we only need if we have flags, which we don't here. I am not sure how that one works though.